### PR TITLE
BAH-2934 | Add EventOffsetMarker scheduler for lite distribution 

### PR DESCRIPTION
**Pull request by Arjun-Go**
[JIRA link](https://bahmni.atlassian.net/browse/BAH-2934?atlOrigin=eyJpIjoiOTAwYTIyYTdhNDYyNDFhN2FiY2YwMmYwY2RlNTNmYWQiLCJwIjoiaiJ9)

# Summary

This PR has changes to add EventOffsetMarker scheduler to lite distributions of Bahmni.

### CODE REVIEW

**Review by SooryaKumaranC-tw — APPROVED** [review]

**Review by deepthi-mantena — APPROVED** [review]

**Review by angshu — APPROVED** [review]

File: masterdata/configuration/liquibase/liquibase.xml [hunk 1]
```diff
@@ -342,4 +342,21 @@
...
+                SELECT COUNT(*) FROM scheduler_task_config where schedulable_class = 'org.openmrs.module.atomfeed.scheduler.tasks.EventRecordsNumberOffsetMarkerTask'
+            </sqlCheck>
+        </preConditions>
+        <comment>Adding scheduler job for bahmni atomfeed offest marker</comment>
+        <sql>
+            INSERT INTO  scheduler_task_config  (name, description, schedulable_class,
+            start_time, start_time_pattern, repeat_interval, start_on_startup, started,
+            created_by, date_created, changed_by, date_changed, last_execution_time, uuid )
+            VALUES ('OpenMRS event offset marker task', NULL, 'org.openmrs.module.atomfeed.scheduler.tasks.EventRecordsNumberOffsetMarkerTask',
+            '2014-01-14 00:00:00','MM/dd/yyyy HH:mm:ss',86400, 1, 1,
```
Comment (gsluthra) [inline masterdata/configuration/liquibase/liquibase.xml:358]: Why was date hardcoded to 2014, and not today()? It is supposed to be `date_created` field. Isn't it? The idea of this column is to know when a row was added to the DB. Putting 2014 seems counter-intuitive. cc: @Arjun-Go

Comment (Arjun-Go) [reply masterdata/configuration/liquibase/liquibase.xml:358]: I went with the same date as the changeSet that was brought in from this [liquibase](https://github.com/Bahmni/clinic-config/blob/main/openmrs/migrations/liquibase.xml#L10) file was also setting `date created` as 2014-01-14. But yes, the value turns out to be counter-intuitive. Will create a fix branch and make this change.

Comment (Arjun-Go) [reply masterdata/configuration/liquibase/liquibase.xml:358]: PR to change date_created field - https://github.com/Bahmni/clinic-config/pull/195

**Review by gsluthra — COMMENTED** [review]
Looks good to me. thanks.

**Pull request merged**